scripts/release_info_test.py

```diff
@@ -352,19 +381,32 @@ def test_check_prs_for_current_release_are_released_with_unreleased_prs(
             requester='', headers='',
             attributes={
                 'title': 'PR1', 'number': 1, 'labels': [
-                    {'name': 'PR: for current release'}]}, completed='')
+                    {'name': release_constants.LABEL_FOR_CURRENT_RELEASE_PRS}]},
+            completed='')
         pull2 = github.PullRequest.PullRequest(
             requester='', headers='',
             attributes={
                 'title': 'PR2', 'number': 2, 'labels': [
-                    {'name': 'PR: released'},
-                    {'name': 'PR: for current release'}]}, completed='')
+                    {'name': release_constants.LABEL_FOR_RELEASED_PRS},
+                    {'name': release_constants.LABEL_FOR_CURRENT_RELEASE_PRS}]},
+            completed='')
+        label = github.Label.Label(
+            requester='', headers='',
+            attributes={
+                'name': release_constants.LABEL_FOR_CURRENT_RELEASE_PRS},
+            completed='')
         # pylint: disable=unused-argument
```

Comment:
Try to remove this by prefixing with unused_ where necessary.

Reply:
Same case as in the reply [here](https://github.com/oppia/oppia/pull/7779#discussion_r337943556).